specification/cognitiveservices/OpenAI.Inference/models/completions/chat_messages.tsp

```diff
@@ -188,6 +221,10 @@ model ChatResponseMessage {
   @doc("The chat role associated with the message.")
   role: ChatRole;
 
+  @added(ServiceApiVersions.v2024_08_01_Preview)
+  @doc("The refusal message generated by the model.")
+  refusal: string;
```

Comment:
Just confirmed that this is actuallyy `nullable` in both Azure and non-Azure OpenAI:

https://github.com/openai/openai-openapi/blob/2ffa9e383cd36ef43a30f766ab25c57d5cadfbbe/openapi.yaml#L9563C14-L9563C16